src/Command/Auth/VerifyPhoneNumberCommand.php

```diff
@@ -105,11 +105,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
         });
 
         $this->debug('Refreshing phone verification status');
-        $needsVerify = $httpClient->post( '/me/phone?force_refresh=1')->json()['verify_phone'];
+        $needsVerify = $httpClient->post( '/me/verification?force_refresh=1')->json();
         $this->stdErr->writeln('');
 
-        if ($needsVerify) {
-            $this->stdErr->writeln('Phone verification succeeded but the status check failed.');
+        if ($needsVerify['state']) {
+            if ($needsVerify['type'] == 'phone') {
+                $this->stdErr->writeln('Phone verification succeeded but the status check failed.');
+            } else if ($needsVerify['type'] == 'ticket') {
```

Comment:
hm, might phone verification have succeeded in this case? if so this command should return `0` (success).. unless we're changing the purpose of the command

Reply:
Yes, this command is specifically about phone verification, so it should ignore the ticket status.